recipes-core/lcd-tools/lcd-tools/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.21)
+project(lcd-tools VERSION 0.1)
+
+set(CMAKE_CXX_STANDARD 20)
+
+add_subdirectory(third-party/cxxopts)
+
+find_package(PkgConfig REQUIRED)
+pkg_search_module(LIBHYBRIS REQUIRED hybris-egl-platform)
```

Comment:
Mhh isn't there a better package than hybris-egl-platform ? Technically you don't need a dependency on the EGL stuff, just the underlying libhybris.

Reply:
Indeed there should be, but I have no idea which it is. There is no `common` target, or other obvious targets for me.

Reply:
Meh... Indeed. Sucks but would need to be fixed upstream